RecoTracker/LST/interface/LSTOutput.h

```diff
@@ -0,0 +1,36 @@
+#ifndef RecoTracker_LST_LSTOutput_h
+#define RecoTracker_LST_LSTOutput_h
+
+#include <memory>
+#include <vector>
+
+class LSTOutput {
+public:
+  LSTOutput() = default;
+  ~LSTOutput() = default;
+
+  enum LSTTCType { T5 = 4, pT3 = 5, pT5 = 7, pLS = 8 };
+
+  void setLSTOutputTraits(std::vector<std::vector<unsigned int>> hitIdx,
+                          std::vector<unsigned int> len,
+                          std::vector<int> seedIdx,
+                          std::vector<short> trackCandidateType) {
+    hitIdx_ = hitIdx;
```

Comment:
Here and elsewhere,
```suggestion
    hitIdx_ = std::move(hitIdx);
```
would be better.